spyder/plugins/editor/plugin.py

```diff
@@ -73,6 +71,7 @@
 from spyder.plugins.toolbar.api import ApplicationToolbars
 from spyder.utils.stylesheet import AppStyle
 from spyder.widgets.mixins import BaseEditMixin
+from spyder.widgets.printer import SpyderPrinter, SpyderPrintPreviewDialog
```

Comment:
I moved the `printer` module to `spyder/widgets` because I needed to use `SpyderPrinter` in the `Print` action of the IPython console.